xbmc/windowing/android/AndroidUtils.cpp

```diff
@@ -406,16 +405,20 @@ CHDRCapabilities CAndroidUtils::GetDisplayHDRCapabilities()
   CHDRCapabilities caps;
   const std::vector<int> types = GetDisplaySupportedHdrTypes();
 
-  if (std::find(types.begin(), types.end(), CAndroidUtils::HDRTypes::HDR10) != types.end())
+  if (std::find(types.begin(), types.end(), CJNIDisplayHdrCapabilities::HDR_TYPE_HDR10) !=
+      types.end())
     caps.SetHDR10();
 
-  if (std::find(types.begin(), types.end(), CAndroidUtils::HDRTypes::HLG) != types.end())
+  if (std::find(types.begin(), types.end(), CJNIDisplayHdrCapabilities::HDR_TYPE_HLG) !=
+      types.end())
     caps.SetHLG();
 
-  if (std::find(types.begin(), types.end(), CAndroidUtils::HDRTypes::HDR10_PLUS) != types.end())
+  if (std::find(types.begin(), types.end(), CJNIDisplayHdrCapabilities::HDR_TYPE_HDR10_PLUS) !=
+      types.end())
     caps.SetHDR10Plus();
 
-  if (std::find(types.begin(), types.end(), CAndroidUtils::HDRTypes::DOLBY_VISION) != types.end())
+  if (std::find(types.begin(), types.end(), CJNIDisplayHdrCapabilities::HDR_TYPE_DOLBY_VISION) !=
+      types.end())
     caps.SetDolbyVision();
```

Comment:
Would it make sense to change `GetDisplaySupportedHdrTypes()` return type to a container with direct access (like a std::set) to avoid the std::find's here?

Reply:
The API method returns a `std::vector`, it could be converted to `std::set` but we would still need an iterator to call the different `set` of the `CHDRCapabilities` object.

In the end there is no clear advantage, several changes would have to be made and I prefer to touch the essentials in this PR in order to use the new constants.

Reply:
Up to you.